test/ansible-operator/cmd/ansible-operator/main.go

```diff
@@ -66,7 +67,7 @@ func main() {
 	}
 
 	// start the operator
-	go operator.Run(done, mgr, "/opt/ansible/watches.yaml")
+	go operator.Run(done, mgr, "/opt/ansible/watches.yaml", 5*time.Second)
```

Comment:
Should this be one minute, to match what `up local` does? I also thought we'd generally settled on a minute as roughly the right default period.